plugins/org.locationtech.udig.style.advanced.feature/src/org/locationtech/udig/style/advanced/lines/LinePropertiesComposite.java

```diff
@@ -417,6 +417,8 @@ public void onStyleChanged( Object source, String[] values, boolean fromField, S
                 } catch (Exception e) {
                     e.printStackTrace();
                 }
+            } else {
+            	ruleWrapper.getRule().setFilter(null);
```

Comment:
Hey :) Can you please apply the code-formatter of uDig to all changed files?

Reply:
Hey, shure - done ;)